api/dps/server.go

```diff
@@ -225,3 +307,70 @@ func (s *Server) ListTransactionsForHeight(_ context.Context, req *ListTransacti
 
 	return &res, nil
 }
+
+// GetResult implements the `GetResult` method of the generated GRPC
+// server.
+func (s *Server) GetResult(_ context.Context, req *GetResultRequest) (*GetResultResponse, error) {
+	txID := flow.HashToID(req.TransactionID)
+
+	result, err := s.index.Result(txID)
+	if err != nil {
+		return nil, fmt.Errorf("could not retrieve transaction result: %w", err)
+	}
+
+	data, err := s.codec.Marshal(result)
+	if err != nil {
+		return nil, fmt.Errorf("could not encode transaction result: %w", err)
+	}
+
+	res := GetResultResponse{
+		TransactionID: req.TransactionID,
+		Data:          data,
+	}
+
+	return &res, nil
+}
+
+// GetSeal implements the `GetSeal` method of the generated GRPC
+// server.
+func (s *Server) GetSeal(_ context.Context, req *GetSealRequest) (*GetSealResponse, error) {
+	sealID := flow.HashToID(req.SealID)
+
+	seal, err := s.index.Seal(sealID)
+	if err != nil {
+		return nil, fmt.Errorf("could not retrieve seal: %w", err)
+	}
+
+	data, err := s.codec.Marshal(seal)
+	if err != nil {
+		return nil, fmt.Errorf("could not encode seal: %w", err)
+	}
+
+	res := GetSealResponse{
+		SealID: req.SealID,
+		Data:   data,
+	}
+
+	return &res, nil
+}
+
+// ListSealsForHeight implements the `ListSealsForHeight` method of the generated GRPC
+// server.
+func (s *Server) ListSealsForHeight(_ context.Context, req *ListSealsForHeightRequest) (*ListSealsForHeightResponse, error) {
+	sealIDs, err := s.index.SealsByHeight(req.Height)
+	if err != nil {
+		return nil, fmt.Errorf("could not list seals by height: %w", err)
+	}
+
+	sIDs := make([][]byte, 0, len(sealIDs))
+	for _, sealID := range sealIDs {
+		sIDs = append(sIDs, sealID[:])
```

Comment:
Could be a good idea yeah 🤔 I'll make a commit without it for now and with manual fixes and will give it a try tomorrow!

Reply:
To be consistent with the Flow funcs I called it `IDToHash` and also made a `CommitToHash` in case we need it for commits as well. Not sure about the `PathToBytes`, while it's inconsistent, in the way it's used it makes more sense than to call it a hash, so I guess it should remain the way it is.